internal/preflight/preflight.go

```diff
@@ -153,3 +176,31 @@ func (p *preflightHelper) verifyBuild(ctx context.Context,
 	}
 	return false, "Waiting for build verification"
 }
+
+func (p *preflightHelper) verifySign(ctx context.Context,
+	pv *kmmv1beta1.PreflightValidation,
+	mapping *kmmv1beta1.KernelMapping,
+	mod *kmmv1beta1.Module) (bool, string) {
+	log := ctrlruntime.LoggerFrom(ctx)
+
+	previousImage := ""
+	if module.ShouldBeBuilt(mod.Spec, *mapping) {
+		previousImage = module.IntermediateImageName(mod.Name, mod.Namespace, mapping.ContainerImage)
+	}
+
+	// at this stage we know that eiher mapping Sign or Container sign are defined
+	signRes, err := p.signAPI.Sync(ctx, *mod, *mapping, pv.Spec.KernelVersion, previousImage, pv.Spec.PushBuiltImage, pv)
+	if err != nil {
+		return false, fmt.Sprintf("Failed to verify signing for module %s, kernel version %s, error %s", mod.Name, pv.Spec.KernelVersion, err)
+	}
+
+	if signRes.Status == utils.StatusCompleted {
+		msg := "sign completes"
+		if pv.Spec.PushBuiltImage {
+			msg += " and image pushed"
```

Comment:
This doesn't seems right? Why are we adding "build" info in the `sign` section? Can it be added elsewhere?

Reply:
not sure i understand what you are talking about... 
this one: pv.Spec.PushBuiltImage?